app/assets/javascripts/modules/_admin.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Admin module
+ *
+ */
+var admin = (function() {
+
+  return {
+    
+    handleTableLinks: function() {
```

Comment:
If I'm understanding correctly (its getting pretty late over here) this javascript handles the linking from table elements to their views? What is the reasoning behind this strategy? Why not use a rails `link_to` within the view itself. I'm not what is causing it, but every 4th time or so that I loaded up a view with a table, the links did not work and i needed to reload